sources/dicom/test_dicom/web_client_specs/dicomWebSpec.js

```diff
@@ -103,8 +103,8 @@ describe('DICOMWeb assetstore', function () {
 
         runs(function () {
             // Set dest type and dest id
-            $('#g-dwas-import-dest-type').val(parentType);
-            $('#g-dwas-import-dest-id').val(parentId);
+            $('#g-dwas-import-dest-type').val(destinationType);
+            $('#g-dwas-import-dest-id').val(destinationId);
```

Comment:
Because we are switching to autoDescribeRoute, some of the tests beyond here will fail.  Specifically, when checking that a float value produces an invalid limit (line 115 if it hasn't changed), we should probably check `return $('.g-validation-failed-message').html().includes('Invalid value');` rather than the `=== 'Invalid limit'`.

Reply:
Would this also apply to line 128 as well?

Reply:
No -- the autoDescribeRoute validates that it is an integer (so its error message applies), but not that it is positive (so the `Invalid Limit` message applies.